--- p4lang-p4c_1.2.4.2-3.dsc
+++ p4lang-p4c_1.2.4.2-3.dsc
@@ -12,11 +12,8 @@
 Package-List:
  p4lang-p4c deb devel optional arch=amd64,arm64
-Checksums-Sha1:
- 5e4431c76e77ead4a6c1eb184d06b20fb9fa4b35 173761028 p4lang-p4c_1.2.4.2.orig.tar.gz
- 6b6eabcfa411decc2e771d6d26ccbd2f29652950 2796 p4lang-p4c_1.2.4.2-3.debian.tar.xz
 Checksums-Sha256:
- 312e94e556d702823e70a3ac0999e17f1de5f1cdf6b444dab2058f562c8313df 173761028 p4lang-p4c_1.2.4.2.orig.tar.gz
+ a41d616add63924a098edaab446809f543ffadfc2e40356f9b73986f324c668a 173758549 p4lang-p4c_1.2.4.2.orig.tar.gz
  2a39bf04c8d084724014b71cdeb3b91dbe7a32bf1bd6cd499e5e132aa3e4cb40 2796 p4lang-p4c_1.2.4.2-3.debian.tar.xz
 Files:
- ead3b9a48dd0052016d69f31df9385f6 173761028 p4lang-p4c_1.2.4.2.orig.tar.gz
+ 98b03592b7a6cf4ed2150463fd757e0e 173758549 p4lang-p4c_1.2.4.2.orig.tar.gz
  f15bf03afd4766e5c986f5036c38e773 2796 p4lang-p4c_1.2.4.2-3.debian.tar.xz
